rpcserver.go

```diff
@@ -5739,7 +5739,18 @@ func (r *rpcServer) GetTransactions(ctx context.Context,
 		return nil, err
 	}
 
-	return lnrpc.RPCTransactionDetails(transactions), nil
+	txDetails := lnrpc.RPCTransactionDetails(transactions)
+
+	if !req.Reverse {
```

Comment:
If we leave the previous logic in `RPCTransactionDetails`, we can do the opposite sorting here only when we need it

```
txDetails := lnrpc.RPCTransactionDetails(transactions)
if !req.Reverse {
        for i, j := 0, len(txDetails)-1; i < j; i, j = i+1, j-1 {
                txDetails[i], txDetails[j] = txDetails[j], txDetails[i]
        }
}

```

Reply:
Okay thank you. I will use thise